features/lwipstack/LWIPInterface.cpp

```diff
@@ -212,19 +213,22 @@ void LWIP::Interface::netif_status_irq(struct netif *netif)
             dns_addr_has_to_be_added = true;
         }
 #endif
-       if (dns_addr_has_to_be_added && !interface->blocking) {
+        if (dns_addr_has_to_be_added && !interface->blocking) {
```

Comment:
Is this just extra whitespace change?
Does it now line up, or not?

Reply:
Yes, just extra whitespace. It lines up now.